app/react-native/src/bin/storybook-start.js

```diff
@@ -84,6 +85,11 @@ if (!program.skipPackager) {
   }
 
   let cliCommand = 'react-native start';
+
+  if (program.metroConfig) {
+    cliCommand += ` --config ${program.metroConfig}`;
```

Comment:
Is it possible to make it work with haul? This variable is reassigned when using it (see below)